prototypes/process-explorer/dotnet/src/MorganStanley.ComposeUI.ProcessExplorer.Abstractions/Processes/ProcessInfoMonitor.cs

```diff
@@ -189,7 +215,7 @@ public bool CheckIfIsComposeProcess(int processId)
         }
         catch (Exception exception)
         {
-            _logger.ProcessExpected(exception);
+            //_logger.ProcessExpected(exception);
```

Comment:
Is this intentionally commented? If yes, please replace with a comment explaining why the exception is swallowed